### DISCUSSION TITLE
Monitoring-As-Code with Crossplane

### DISCUSSION THREAD

**Discussion by luebken** [crossplane/crossplane#3998] [General]
I've put a little piece together that explores building up a monitoring platform with Crossplane: https://blog.crossplane.io/monitoring-as-code-with-crossplane/. As noted in the article, this is supposed to be a discussion starter, which I'm starting here.

**Comment by jjackson2-legalzoom** [comment]
Definitely interested in seeing more of this! We're looking into IaC-ified monitoring systems at the moment, and although CW Alarms are trivial to create with CDK, hooking them up to a ticketting system (Jira) is more involved - DataDog Monitors are [_technically_ supported](https://docs.datadoghq.com/serverless/libraries_integrations/cdk/), but unergonomic; and I haven't yet delved into [using OpsItems are an intermediate](https://aws.amazon.com/blogs/mt/use-jira-service-management-to-track-and-resolve-operational-items-related-to-aws-resources/). Would love to see another alternative!
